### ISSUE TITLE
train error

### ISSUE THREAD

**Issue by songwei100110** [chengwei-zheng/EditableNeRF_cvpr23#1]

**Comment by chengwei-zheng** [comment]
Hi, this is a bug caused by the config file, you can change the `NerfModel.load_kp_init_file = 'k_points_auto_2p.txt'` in `editablenerf_2p.gin` to `NerfModel.load_kp_init_file = 'k_points_auto.txt'`, or use the one I [commit](https://github.com/chengwei-zheng/EditableNeRF_cvpr23/commit/8ae073f8a21ce9993e2649f50e543a7d89be46bf) just now.

**Comment by songwei100110** [comment]
I have debugged this code of yours before, but when I use the data set capture_dice_cup.zip provided by you, my parameter Settings are as follows, and the program can also run through, I don't know whether the parameter Settings are correct?
![image](https://github.com/chengwei-zheng/EditableNeRF_cvpr23/assets/44295578/a0a9fcad-b738-4fa3-961f-8111c93a3a9b)
In addition, I tried to install the environment package of pip you provided, but the gpu could not be used directly, and finally I used the cpu to run through.
Finally, thank you for your reply！

**Comment by chengwei-zheng** [comment]
Yes, the settings are correct. Regarding the environment, it may be a problem with JAX. Have you Installed JAX as in [HyperNeRF](https://github.com/google/hyperNeRF)? And you can also run HyperNeRF to check the environment.

**Comment by songwei100110** [comment]
ok, thank you. 
thank you for your reply！

**Comment by songwei100110** [comment]
![image](https://github.com/chengwei-zheng/EditableNeRF_cvpr23/assets/44295578/4f75d7cb-4c12-4caf-9e1b-3718b45b5f4e)
1. Do you get your dataset using the data processing script from https://github.com/google/nerfies/blob/main/notebooks/Nerfies_Capture_Processing.ipynb?
2. Can you share the original video of your dataset? Or the original video format of capture_dice_cup. I want to learn.
thank you!

**Comment by chengwei-zheng** [comment]
1. Yes. Two minor differences are that a) we skip the step to flatten the video into images and b) we use COLMAP with GUI to check the results.
2. Images in `capture_dice_cup/rgb/1x/` is actually the original inputs. We didn't transform them into video. You can copy them into `rgb-raw` folder as in HyperNeRF if you want to process it yourself.

**Comment by songwei100110** [comment]
ok, thank you.

**Comment by songwei100110** [comment]
![Uploading image.png…]()

The data involved in these two evaluation tables are not open source in your provided dataset?

**Comment by songwei100110** [comment]
The data involved in these two evaluation tables including Tab1 and Tab2 are not open source in your provided dataset?

**Comment by chengwei-zheng** [comment]
Thanks for your notice. This is an issue due to the storage limitation of Google Drive, we will upload them later.

**Comment by songwei100110** [comment]
Okay, can you share it with me in advance? I want to see the effect. thank you.My email: songwei100110@gmail.com

**Comment by chengwei-zheng** [comment]
Some sequences have been additionally uploaded to EditableNeRF_data/dataset. Please find them [here](https://github.com/chengwei-zheng/EditableNeRF_cvpr23#dataset)

**Comment by songwei100110** [comment]
![image](https://github.com/chengwei-zheng/EditableNeRF_cvpr23/assets/44295578/561e839d-002f-45e1-8906-48831af7ec6d)

This error occurred when I ran your GUI_qt.py?

**Comment by songwei100110** [comment]
![image](https://github.com/chengwei-zheng/EditableNeRF_cvpr23/assets/44295578/78fa0aa6-2a7b-4dfe-947c-97f0509848df)
You set it to image_scale=8 by default, but I found that the value of this variable in hypernerf is 4, so I set it to 4, is that right?

**Comment by chengwei-zheng** [comment]
The current GUI only supports scenes containing one keypoint. If you want to use GUI, you can try it on `dataset/capture_drawer.zip`.
For multi-keypoint scenes, you can edit them following the comments in `encode_metadata()` of [evaluation.py](https://github.com/chengwei-zheng/EditableNeRF_cvpr23/blob/main/editablenerf/evaluation.py)

**Comment by songwei100110** [comment]
hi, your ablation experiment shown on table3 is executed on capture_drawer?

**Comment by chengwei-zheng** [comment]
No, it is on the synthetic data, including `syn_drawer`.

**Comment by songwei100110** [comment]
![image](https://github.com/chengwei-zheng/EditableNeRF_cvpr23/assets/44295578/ac705dbd-2959-4e4e-9c2b-a818ecbef5dc)
When I used the data capture_dice_cup provided by you and visualized its point cloud created by colmap and initial key points in the first frame of the image. I found that the distribution of key points seemed to be inaccurate. Then I would like to ask, will the initial key points be distributed near the two key points at the beginning?

**Comment by chengwei-zheng** [comment]
Hi, the key point is in a different coordinate system with additional transformation and scalar. You can refer to the **Local coordinate systems for key points** in **Sec.1 Additional Details** from our [Supplementary](https://chengwei-zheng.github.io/EditableNeRF/EditableNeRF_supplementary_cvpr23.pdf).

**Comment by songwei100110** [comment]
Ok, I'll look into it carefully. Thank you for your reply.
